cloudflare/resource_cloudflare_load_balancer_monitor.go

```diff
@@ -210,14 +246,54 @@ func resourceCloudflareLoadBalancerPoolMonitorUpdate(d *schema.ResourceData, met
 		loadBalancerMonitor.Port = uint16(port.(int))
 	}
 
-	if allowInsecure, ok := d.GetOk("allow_insecure"); ok {
-		loadBalancerMonitor.AllowInsecure = allowInsecure.(bool)
-	}
+	switch loadBalancerMonitor.Type {
+	case "tcp":
+		if method, ok := d.GetOk("method"); ok {
+			loadBalancerMonitor.Method = method.(string)
+		} else {
+			loadBalancerMonitor.Method = "connection_established"
```

Comment:
This isn't a HTTP verb. Is it intended?

Reply:
Yes, it's only valid when the "type" is "tcp" and this block is only is in the swtich/case "tcp" statement.